pkgs/by-name/mo/moneydance/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, stdenv, fetchurl, makeWrapper, openjdk21, openjfx21, }:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "moneydance";
+  version = "2023.3_5064";
+
+  src = fetchurl {
+    url = "https://infinitekind.com/stabledl/${finalAttrs.version}/moneydance-linux.tar.gz";
+    hash = "sha256-liU47bTKGBhDCnEQaKA0rUNPmBYH/zMJ7FOhmt5hQ14=";
+  };
+
+  jdk = openjdk21.override { enableJavaFX = true; };
```

Comment:
I'm not sure I quite like passing `jdk` as an argument to `mkDerivation`.

IMO it should be defined in a `let` block and `inherit`ed as `passthru.jdk` if you want to refer to it in other parts of the package definition.

Reply:
Ah yes, that's good; I'm learning things about derivations today (I'm quite new to this). And that's not a totally theoretical benefit, since Moneydance supports plugins written in Java.